addons/beehave/nodes/leaves/blackboard_compare.gd

```diff
@@ -0,0 +1,39 @@
+@tool
+class_name BlackboardCompareCondition extends ConditionLeaf
+
+
+enum Operators {
+	EQUAL,
+	NOT_EQUAL,
+	GREATER,
+	LESS,
+	GREATER_EQUAL,
+	LESS_EQUAL,
+}
+
+
+@export var left_operand: String = ""
+@export_enum("==", "!=", ">", "<", ">=", "<=") var operator: int = 0
+@export var right_operand: String = ""
+
+
+@onready var _left_expression: Expression = parse_expression(left_operand)
+@onready var _right_expression: Expression = parse_expression(right_operand)
+
+
+func tick(actor: Node, blackboard: Blackboard) -> int:
+	var left: Variant = _left_expression.execute([], blackboard)
+	var right: Variant = _right_expression.execute([], blackboard)
```

Comment:
I think we could check for `has_execute_failed` here like in the `blackboard_set.gd`.